src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo__img {
+  display: flex;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). It's better to style elements using classes. Instead of targeting 'ul' directly for .nav__list, define a class that can be used for this purpose.